client/packages/requisitions/src/RequestRequisition/DetailView/columns.ts

```diff
@@ -118,7 +118,6 @@ export const useRequestColumns = () => {
       label: 'label.requested-packs',
       description: 'label.requested-number-packs',
       align: ColumnAlign.Right,
-      // width: 150,
```

Comment:
This was left from previous, thought it better to remove rather than leave commented?

Reply:
Better to remove I think